src/odemis/gui/xmlh/resources/panel_fastem_user_settings.xrc

```diff
@@ -107,85 +107,14 @@
-            <object class="sizeritem">
-              <object class="ImageButton" name="btn_log">
-                <icon>../../img/icon/ico_chevron_up.png</icon>
-                <height>16</height>
-                <face_colour>def</face_colour>
-                <tooltip>Open log panel</tooltip>
-                <style>wxALIGN_CENTRE</style>
-                <XRCED>
-                  <assign_var>1</assign_var>
-                </XRCED>
-              </object>
-            </object>
           </object>
-          <size>300,-1</size>
           <bg>#333333</bg>
         </object>
-        <flag>wxALL|wxEXPAND</flag>
-        <border>10</border>
-      </object>
-      <object class="sizeritem">
-        <object class="ARAcquiViewport" name="vp_chamber">
-          <XRCED>
-            <assign_var>1</assign_var>
-          </XRCED>
-        </object>
-        <option>1</option>
-        <flag>wxEXPAND</flag>
-      </object>
-      <object class="sizeritem">
-        <object class="wxPanel">
-          <size>400,-1</size>
-          <bg>#333333</bg>
-          <style>wxBORDER_NONE</style>
-          <object class="wxBoxSizer">
-            <object class="sizeritem">
-              <object class="wxScrolledWindow" name="scr_win_right">
-                <object class="wxBoxSizer">
-                  <orient>wxVERTICAL</orient>
-                  <object class="sizeritem">
-                    <object class="FoldPanelBar">
-                      <object class="FoldPanelItem">
-                        <object class="StreamBar" name="pnl_streams">
-                          <size>300,-1</size>
-                          <fg>#7F7F7F</fg>
-                          <bg>#333333</bg>
-                          <XRCED>
-                            <assign_var>1</assign_var>
-                          </XRCED>
-                        </object>
-                        <label>OPTICAL</label>
-                        <fg>#1A1A1A</fg>
-                        <bg>#555555</bg>
-                      </object>
-                      <spacing>0</spacing>
-                      <leftspacing>0</leftspacing>
-                      <rightspacing>0</rightspacing>
-                      <bg>#333333</bg>
-                    </object>
-                    <flag>wxEXPAND</flag>
-                  </object>
-                </object>
-                <size>400,-1</size>
-                <bg>#333333</bg>
-                <style>wxVSCROLL</style>
-                <XRCED>
-                  <assign_var>1</assign_var>
-                </XRCED>
-              </object>
-              <option>1</option>
-              <flag>wxEXPAND</flag>
-              <minsize>400,400</minsize>
-            </object>
-            <orient>wxVERTICAL</orient>
-          </object>
-        </object>
         <flag>wxEXPAND</flag>
+        <border>10</border>
       </object>
-      <orient>wxHORIZONTAL</orient>
     </object>
     <bg>#333333</bg>
+    <size>300,600</size>
```

Comment:
@pieleric `<size>300,-1</size>` is not working, the panel is cropped in `y`. Do you know what could be the reason unlike previously when it worked?

Reply:
It should be related to the sizer. I'm not sure what, it could be related to the proportions of the other elements in the sizer, or maybe some flag.

Reply:
I managed to solve this in this commit https://github.com/delmic/odemis/pull/2649/commits/a109561e02d75ca0a6e93d1d9b5356d4e1555af4